packages/components/src/Button/button.stories.tsx

```diff
@@ -13,30 +33,48 @@ export default {
         type: "text",
       },
     },
+    color: {
```

Comment:
for some reason these weren't showing up 😞  it's weird b/c the other Button props are declared similarly, but show up correctly